src/ARMS/chassis.cpp

```diff
@@ -209,7 +213,7 @@ void velocity(int t, int max) {
 void arc(bool mirror, int arc_length, double rad, int max, int type) {
 	reset();
 	int time_step = 0;
-	chassisMode = 0;
+	mode = NON_PID_MODE;
```

Comment:
The mode variable only applies to the `chassisTask`, which handles the PID. So PID is implied, and since the variable is called `mode` we can remove mode from the enum name. This would allow us to be able shorten the enum names to:
LINEAR
ANGULAR
DISABLE